### PR TITLE
More comfortable auto generate combined num

### PR DESCRIPTION
**Pull request by rocketkai**
Review @jeremylim91's work

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,24 +56,28 @@ <h1 id="header">Basics: Beat That! 🚀</h1>
       <input id="input-field" />
       <button id="submit-button">Submit</button>
       <p>Output:</p>
-      <div id="output-div"></div>
+      <div id="output-div">
```
Comment (kai-rocket) [inline index.html:59]: @jeremylim91 could you help to rebase on top of `main` to avoid these changes? thank you!

File: script.js [hunk 2]
```diff
@@ -1,4 +1,368 @@
...
+
+// Use player profiles to keep track of each player's details
+var playerProfiles = [
+  {id: 1, score: 0},
+  {id: 2, score: 0},
+];
+
+/*====================================
+==============HELPER FUNCTIONS========
+======================================*/
```
Comment (kai-rocket) [inline script.js:43]: @jeremylim91 helper functions isn't a good header. all functions are helper functions except for `main`. could we either remove this or rename it?

File: script.js [hunk 3]
```diff
@@ -1,4 +1,368 @@
...
+  } else {
+    player2Num = playerNum;
+  }
+};
+
+/**
+ * Automate choosing the player's number from the dice rolls
+ * Return the player number
+ * @param {number} firstNumeralIndex
+ */
```
Comment (kai-rocket) [inline script.js:167]: @jeremylim91 what's this? Let's not leave dangling comments- this is confusing and not professional

File: script.js [hunk 4]
```diff
@@ -1,4 +1,368 @@
...
+    return 1;
+  }
+  // Else if in lowest combined number mode and player 1 has a smaller number, player 1 wins
+  else if (mode == LOWEST_COMBINED_NUMBER && player1Num < player2Num) {
+    return 1;
+  }
+  // in all other scenarios (even if draw), return that player 2 has won
+  return 2;
+};
+/**
```
Comment (kai-rocket) [inline script.js:186]: @jeremylim91 let's be consistent and leave newlines between functions

File: script.js [hunk 5]
```diff
@@ -1,4 +1,368 @@
...
+  currPlayer = 1;
+  gameMode = GAME_MODE_DICE_ROLL;
+};
+
+/*====================================
+==============MAIN====================
+======================================*/
+/**
+ * Play Beat That as per SWE101 game rules
+ * https://swe101.rocketacademy.co/projects/project-2-dice
```
Comment (kai-rocket) [inline script.js:275]: @jeremylim91 let's rename SWE101 to Coding Basics everywhere 🙏

File: script.js [hunk 6]
```diff
@@ -1,4 +1,368 @@
...
+ */
+var resetGame = function () {
+  currPlayer = 1;
+  gameMode = GAME_MODE_DICE_ROLL;
+};
+
+/*====================================
+==============MAIN====================
+======================================*/
+/**
```
Comment (kai-rocket) [inline script.js:273]: Is this section header necessary?

File: script.js [hunk 7]
```diff
@@ -1,4 +1,368 @@
...
+      currPlayer +
+      '. <br>You rolled Dice 1: ' +
+      newDiceRolls[0] +
+      ' and Dice 2: ' +
+      newDiceRolls[1] +
+      '<br>    Click submit to see your number'
+    );
+  }
+
+  //Automatically generate the largest possible number based on a player's dice roll
```
Comment (kai-rocket) [inline script.js:319]: Space after `//` for consistency

File: script.js [hunk 8]
```diff
@@ -1,4 +1,368 @@
...
+    if (currPlayer == 1) {
+      currPlayer = 2;
+      gameMode = GAME_MODE_DICE_ROLL;
+      // Return player number to Player 1, let Player 2 know it is their turn
+      return (
+        playerNumResponse +
+        " <br> It is now Player 2's turn. Press Submit to roll Player 2's dice."
+      );
+    }
+    // // Else if currPlayer is Player 2, determine the winner and let the players know who won.
```
Comment (kai-rocket) [inline script.js:342]: Double `//` unnecessary

File: script.js (hunk 7 above)
Comment (kai-rocket) [reply script.js:319]: @jeremylim91 are you using prettier when coding? thought prettier would have fixed this automatically

**Review by rocketkai — COMMENTED** [review]
Mostly formatting. Thanks for hard work!

**Pull request closed without merging**